README.md

```diff
@@ -1,2 +1,17 @@
 # dataverse-language-packs
 Repository for language files associated with Dataverse
+
+Some language packages supported by Dataverse version not older than 4.9.4 (please see notes below)
+
+Available languages:
+- English
+- [French (Canada)](https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/tree/develop/fr_CA) maintained by [Scholars Portal](https://dataverse.scholarsportal.info)
```

Comment:
French (Canada) translation is done by Bibliothèques Université de Montréal, in collaboration with Scholars Portal. If anyone wants to comment about/suggest translations to us.

Reply:
sounds good, @4tikhonov can you add this statement?